lambdas/build-cri-oauth-request/src/test/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandlerTest.java

```diff
@@ -112,6 +117,18 @@ class BuildCriOauthRequestHandlerTest {
     private static final String JOURNEY_BASE_URL = "/journey/cri/build-oauth-request/";
     private static final String TEST_EMAIL_ADDRESS = "test@test.com";
     private static final String TEST_NI_NUMBER = "AA000003D";
+    private static final String CONTEXT = "context";
+    private static final String BANK_ACCOUNT_CONTEXT = "context";
```

Comment:
I suppose it doesn't matter, but should this be `bank_account`?

Alternatively, should it just be `TEST_CONTEXT` (any reason why it should be bank account related?)